src/index.js

```diff
@@ -0,0 +1,167 @@
+                params: {
+                    lat: latitude,
+                    lon: longitude
+                }
+            })
+                .then((response) => {
+                    const degreesKelvin = response.data.main.temp
+                    const degreesFahrenheit = Math.floor(((degreesKelvin - 273.15) * (9 / 5) + 32));
+                    state.degrees = degreesFahrenheit;
+                    document.querySelector('#degrees').textContent = state.degrees;
+                })
+                .catch((error) => {
+                    console.log(error)
+                })
+        })
+        .catch((error) => {
+            console.log(error)
+        });
+}
+
+const changeColorTemp = () => {
+    const num = state.degrees
+
+    if (num <= 49) {
+    } else if (num > 49 && num <= 59) {
+        document.getElementById('degrees').style.color = 'green';
+    } else if (num > 59 && num <= 69) {
+        document.getElementById('degrees').style.color = 'gold';
+    } else if (num > 69 && num <= 79) {
+        document.getElementById('degrees').style.color = 'orange';
+    } else {
+        document.getElementById('degrees').style.color = 'red'
+    }
+}
+
+const landscape = {
+    summer: "🌵__🐍_🦂_🌵🌵__🐍_🏜_🦂",
+    spring: "🌸🌿🌼__🌷🌻🌿_☘️🌱_🌻🌷",
+    autumn: "🌾🌾_🍃_🪨__🛤_🌾🌾🌾_🍃",
+    winter: "🌲🌲⛄️🌲⛄️🍂🌲🍁🌲🌲⛄️🍂🌲"
+}
+
+function displayLandscape(degrees) {
+    let currentLandscape = document.getElementById('seasonal-landscape');
+    if (degrees <= 59) {
+        currentLandscape.textContent = landscape.winter;
+    } else if (degrees <= 69) {
+        currentLandscape.textContent = landscape.autumn;
+    } else if (degrees <= 79) {
+        currentLandscape.textContent = landscape.spring;
+    } else {
+        currentLandscape.textContent = landscape.summer;
+    }
+}
+
+const sky = {
+    sunny: "☁️ ☁️ ☁️ ☀️ ☁️ ☁️",
+    cloudy: "☁️☁️ ☁️ ☁️☁️ ☁️ 🌤 ☁️ ☁️☁️",
+    rainy: "🌧🌈⛈🌧🌧💧⛈🌧🌦🌧💧🌧🌧",
+    snowy: "🌨❄️🌨🌨❄️❄️🌨❄️🌨❄️❄️🌨🌨"
+}
+
+const dropDownOutput = document.getElementById("skies");
+
+const displaySky = () => {
+    const selectedVal = dropDownOutput.value;
+
+    if (selectedVal in sky) {
+        const skyLandscape = document.getElementById('sky-landscape');
+        skyLandscape.textContent = sky[selectedVal]
+    }
+
+    if (selectedVal === 'snowy') {
+        document.getElementById("whole-landscape").style.backgroundColor = "#366492";
+    } else if (selectedVal === 'cloudy') {
+        document.getElementById("whole-landscape").style.backgroundColor = "#A1ABB5";
+    } else if (selectedVal === 'rainy') {
+        document.getElementById("whole-landscape").style.backgroundColor = "#60ABF6";
+    } else {
+        document.getElementById("whole-landscape").style.backgroundColor = "#adeded";
+    }
+}
+
+const increaseTemp = () => {
+    state.degrees += 1;
+    const degrees = document.querySelector('#degrees');
+    degrees.textContent = state.degrees;
+    displayLandscape(state.degrees);
```

Comment:
I see this code repeated in L113-1115 -- this could be a good place to make a helper function!